src/expr/src/scalar/func.rs

```diff
@@ -5928,8 +5928,16 @@ fn create_range<'a>(
     datums: &[Datum<'a>],
     temp_storage: &'a RowArena,
 ) -> Result<Datum<'a>, EvalError> {
-    let (lower_inclusive, upper_inclusive) =
-        range::parse_range_bound_flags(datums[2].unwrap_str())?;
+    let flags = match datums[2] {
```

Comment:
I thought NULLs were prevented from even getting here?

Reply:
> I thought NULLs were prevented from even getting here?

Where would this be specified?

Reply:
@mjibson Yeah that was my mistake, as well––this is often the case, but this function was not set to propgate nulls (nor should it, apparently).

@aalexandrov This is defined as `propagates_nulls` on `UnaryFunc`, `BinaryFunc`, `VariadicFunc`.